common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayWorkV2.scala

```diff
@@ -16,9 +16,10 @@ case class DisplayWorkV2(
     readOnly = true,
     description = "The canonical identifier given to a thing.") id: String,
   @Schema(
+    `type` = "String",
```

Comment:
I don't think we need explicitly pass this here, it is derived automatically in the current `swagger.json` as far as I can see

Reply:
I remember we needed to do this when we wrap things into `Option` because it shows a weird type otherwise. But I haven't checked swagger output admittedly. If it doesn't anymore then, all the better

Reply:
It shouldn't be necessary anymore since update to v3 (there is custom `Option[_]` handling [here](https://github.com/wellcometrust/catalogue/blob/master/api/api/src/main/scala/uk/ac/wellcome/platform/api/swagger/SwaggerScalaModelConverter.scala#L38-L39))

Reply:
Nice! thanks